pages/views/index.11tydata.yml

```diff
@@ -0,0 +1,89 @@
+seoTitle: ESL — A Web Components Library
+description: >
+  ESL is an open-source UI component library based on web components.
+  Create user-friendly interfaces and build sites from the ground up faster using framework agnostic web components.
+
+landing:
+  marquee:
+    logo: logo-pretty.svg
+    title: Exadel Smart Library
+    slogan: UI component library with flair and precision
+    description: >
+      ESL is an open-source web components library that helps developers
+      create modern-looking, effective, and user-friendly interfaces faster.
+    ctas:
+      - text: Get started
+        url: /introduction
+        style: sec-blue
+      - text: See the library in action
+        url: /examples
+        style: sec-orange
+
+  benefits:
+    title: Why the ESL Web Components Library?
+    items:
+      - icon: benefits/efficient.svg
+        name: Efficient
+        text: >
+          Our team has reduced the work needed to create reusable UI components by giving you a reliable base
+          to customize and expand on for fast and accurate development.
+      - icon: benefits/fast.svg
+        name: Fast
+        text: >
+          With ESL, there’s no need to write lengthy markup — the reusable elements in our UI component library are small,
+          but they’re designed to be the optimal building blocks for your content.
+      - icon: benefits/light.svg
+        name: Light
+        text: >
+          We specially designed our reusable web components to work quickly and efficiently in browsers,
+          cutting out the unnecessary code that takes a toll on user experience.
```

Comment:
```suggestion
          We specially designed ESL's web components to work quickly and efficiently in browsers,
          cutting out the unnecessary code that takes a toll on user experience.
```

Reply:
We should also cover some SEO reasons. But you're right, in general not all the keywords look natural in the text.
For that phrase let's better leave it as it is.